kolibri/core/errorreports/schemas.py

```diff
@@ -0,0 +1,54 @@
+context_frontend_schema = {
+    "type": "object",
+    "properties": {
+        "browser": {"type": "string"},
+        "component": {"type": "string", "optional": True},
+        "device": {
+            "type": "object",
+            "properties": {
+                "type": {"type": "string"},  # "desktop", "tablet", "mobile"
+                "platform": {
+                    "type": "string",
+                    "optional": True,
+                },  # "windows", "mac", "linux", "android", "ios"
```

Comment:
With json schema we can ensure the platform is from a predefined list of platforms